FF1Blazorizer/Pages/Randomize.razor

```diff
@@ -19,6 +19,7 @@
 	<BugFixesTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
 	<ExperimentalTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
 	<FunTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
+	<HintsTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
```

Comment:
It should probably be put before the Conveniences Tab

Reply:
done